README.md

```diff
@@ -83,6 +83,7 @@
 
 * [bento](https://github.com/rapenne-s/bento/) - A KISS deployment tool to keep your NixOS fleet (servers & workstations) up to date.
 * [Colmena](https://github.com/zhaofengli/colmena) - A simple, stateless NixOS deployment tool modeled after NixOps and morph.
+* [comin](https://github.com/nlewo/comin) - A deployment tool working in pull mode.
```

Comment:
Should definitely mention the main feature part of it: using the Git repos themselves. See this for a suggestion, or make your own:

```suggestion
* [comin](https://github.com/nlewo/comin) - A deployment tool to continuously pull from Git repositories.
```